src/specs/01_basic/multi_tenancy.spec

```diff
@@ -0,0 +1,196 @@
+* `"JPM Investment"` can read "contract1" from "Node1"
+* `"GS Investment"` fails to read "contract1" from "Node1"
+* `"GS Investment"` fails to write a new arbitrary value to "contract1" by sending a transaction to `"Node1"` with its TM key `"GS_K1"` using `"GS_ACC1"` and private for `"GS_K1"`
+* `"GS Investment"` deploys a "SimpleStorageDelegate(contract1)" private contract, named "delegateContract", by sending a transaction to `"Node1"` with its TM key `"GS_K1"` using `"GS_ACC1"` and private for `"GS_K1"`
+* `"GS Investment"` fails to write a new arbitrary value to "delegateContract" by sending a transaction to `"Node1"` with its TM key `"GS_K1"` using `"GS_ACC1"` and private for `"GS_K1"`
+
+## JPM, GS and DB using self-managed account keys can access public states
+
+tags: public, raw
+
+* `"JPM Investment"` deploys a "SimpleStorage" public contract, named "contract1", by sending a transaction to `"Node1"`
+* `"GS Investment"` writes a new value "123" to "contract1" successfully by sending a transaction to `"Node1"`
+* `"DB Investment"` reads the value from "contract1" successfully by sending a request to `"Node2"` and the value returns "123"
+
+## JPM, GS and DB using node-managed account keys can access public states
+
+tags: public, node-managed-account
+
+* `"JPM Investment"` deploys a "SimpleStorage" public contract, named "contract1", by sending a transaction to `"Node1"` using `"JPM_ACC1"`
+* `"GS Settlement"` writes a new value "123" to "contract1" successfully by sending a transaction to `"Node2"` using `"GS_ACC2"`
+* `"DB Investment"` reads the value from "contract1" successfully by sending a request to `"Node2"` and the value returns "123"
+
+## GS using self-managed account keys can NOT bypass access controls to private contracts which are not privy to it by using nonce shifts
+
+tags: private, access, raw
+
+ Tenants who self-manage account keys use raw transaction flow to submit private transactions
+ `GS Investment` can not write to private contracts on which `B1` or `B2` are not participants
+ `GS Investment` fails when attempting to use its private contract that manipulates other private contracts on which `B1` or `B2` are not participants
+
+* `"JPM Investment"` deploys a "SimpleStorage" private contract, named "contract1", by sending a transaction to `"Node1"` with its TM key `"JPM_K1"`, signed by `"Wallet1"` and private for `"JPM_K1"`
+* `"JPM Investment"` writes a new arbitrary value to "contract1" successfully by sending a transaction to `"Node1"` with its TM key `"JPM_K1"`, signed by `"Wallet1"` and private for `"JPM_K1"`
+* `"JPM Investment"` can read "contract1" from "Node1"
+* `"GS Investment"` fails to read "contract1" from "Node1"
+* `"GS Investment"` fails to write a new arbitrary value to "contract1" by sending a transaction to `"Node1"` with its TM key `"GS_K1"`, signed by `"Wallet2"` and private for `"GS_K1"`
```

Comment:
Next scenario test could be added:

```
* `"GS Investment"` fails to write a new arbitrary value to "contract1" by sending a transaction to `"Node1"` with its TM key `"GS_K1"`, signed by `"Wallet2"` and private for `"JPM_K1"`
```

to make sure that even if trying to set PrivateFor to JPM_K1 which is part of the managedParties then the transaction fails